test/expenses.routes.test.js

```diff
@@ -699,7 +708,7 @@ describe('expenses.routes.test.js', () => {
 
                 var expense, transaction;
                 beforeEach(() => expectTwo(Expense).tap(e => expense = e));
-                beforeEach(() => expectOne(Transaction).tap(t => transaction = t));
+                beforeEach(() => expectTwo(Transaction).tap(t => transaction = t));
```

Comment:
nice to have, replace `expectOne` and `expectTwo` with a new `expectAmount` in `utils`